modules/internal/CPtr.chpl

```diff
@@ -106,6 +106,9 @@ module CPtr {
   pragma "no doc"
   inline proc =(ref a:c_ptr, b:c_ptr) { __primitive("=", a, b); }
 
+  pragma "no doc"
+  inline proc =(ref a: c_ptr, b: c_void_ptr) { __primitive("=", a, b); }
+
```

Comment:
How about going in the other direction, `=(c_void_ptr, c_ptr)` ? Seems it is equally - or even more - valid.

Reply:
That should already work because we allow coercions from c_ptr to c_void_ptr.

Reply:
Trying a quick test, it seems it does.